### PR TITLE
Unit test cases fixed - LB, configurator lib files etc

### PR DESCRIPTION
**Pull request by pkharat**

### CODE REVIEW

**Comment by pkharat** [comment]
@dpaks please review test_demuxer.py and fw_test_data.py as I have modified some data dictionaries.

**Comment by njagadish** [comment]
changes are fine except that @dpaks and @pkharat need to come up with common changes for firewall test data. Once it is made common, the unit tests can be merged.

File: gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py [hunk 1]
```diff
@@ -96,17 +98,19 @@ def test_update_pool_stats(self):
             context = test_data.Context()
             agent.update_pool_stats('pool_id', 'stats', context)
 
+            data = {'info': {'service_type': 'loadbalancer',
```
Comment (njagadish) [inline gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py:101]: Is the data supposed to be in the lb_test_data.py file?

File: gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py [hunk 2]
```diff
@@ -10,18 +10,22 @@
...
 
-#import json
-import mock
-import unittest
 
+import mock
 import oslo_serialization.jsonutils as jsonutils
 import pecan
-from pecan import rest
+PECAN_CONFIG_FILE = '/opt/stack/gbp/gbpservice/nfp/configurator/api/config.py'
```
Comment (njagadish) [inline gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py]: Unit tests should run on any machine where code is downloaded. In this case, this path is not suitable right?

File: gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py (hunk 1 above)
Comment (pkharat) [inline gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py:101]: This data is specific to this api but lb_test_data.py has resources data which is used across all apis. So kept it here.

File: gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py (hunk 2 above)
Comment (pkharat) [inline gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py]: Assumed test cases will be run under /opt/stack/gbp/. If it is kept in other location then will have to modify this variable. any other suggestion ?

**Comment by pkharat** [comment]
Closing this as new one is raised on mitaka_nfp_17_may_2016 branch

**Pull request closed without merging**